setup.py

```diff
@@ -11,11 +10,9 @@
     install_requires=[
         "PyYAML>=3.11,<6",
         "elasticsearch>=7.0.0,<8.0.0",
+        "deepdiff~=6.5.0",
+        "mergedeep~=1.3.4",
```

Comment:
please remove restrictions for libraries.

Reply:
This is not just a library. It's used in [elasticsearch](https://github.com/NCI-GDC/tungsten/blob/develop/salt/pillar/services.sls#L147-L148) vms.